docs/locale/ja/LC_MESSAGES/annotations.po

```diff
@@ -0,0 +1,246 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-2018, spotbugs community
+# This file is distributed under the same license as the spotbugs package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: spotbugs 4.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-11-05 12:44+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
```

Comment:
Should probably populate or drop this.

Reply:
It seems that the email address is not configurable. Let's put it aside for now.
https://sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html